### PR TITLE
Add option to disable boss cast bars

### PR DESCRIPTION
**Pull request by spikeh**
I personally find the "<Cast: %s>" bars distracting and only want to see timer bars.

### CODE REVIEW

File: Core/BossPrototype.lua [hunk 1]
```diff
@@ -1464,6 +1464,8 @@ end
 -- @param[opt] icon the bar icon (spell id or texture name)
 function boss:Bar(key, length, text, icon)
 	local textType = type(text)
+	local bars = core:GetPlugin("Bars")
+	if bars and not bars.db.profile["cast"] and text == L.cast then return end
```
Comment (funkydude) [inline Core/BossPrototype.lua]: I'm assuming you haven't tested this because this code will not work.

Comment (spikeh) [inline Core/BossPrototype.lua]: Yeah totally my fault for not checking, I forgot `text` here is after formatting.

How would you handle different locales for L.cast? Would a per-ability per-encounter tickbox option be better than a generic catch-all in boss:Bar?

Comment (funkydude) [inline Core/BossPrototype.lua]: Yes, per-boss options would be required which is why this isn't a simple feature. It requires a new API implementation and propagation across all modules.

Comment (spikeh) [inline Core/BossPrototype.lua]: I'm working on it.

**Pull request closed without merging**